frontend/src/recipeData.js

```diff
@@ -0,0 +1,30 @@
+const recipes = [
+  {
+    "recipeName":"Chicken65",
+    "link":'chicken',
+    "recipeDescription":"\n An immensely popular dish hailing from India features spicy fried chicken cooked with curry leaves, garlic, and chilli peppers. There are different recipes for this dish with varying levels of complexity. This recipe includes all the staple flavors\n ",
+    "recipeImage":"https://1.bp.blogspot.com/-yJBQwqQcUOs/X5gtEJXEoCI/AAAAAAAAY94/gut_UssF__8lT0P2kl9patf4FY-U4YCtQCLcBGAsYHQ/w1200-h630-p-k-no-nu/chicken%2B65%2B4.JPG",
+    "ingredientList":["1 lb chicken breasts, cut into pieces (your preference in size)","1 egg","2 tablespoons cornstarch","1 tablespoon of diced garlic","2 teaspoons of salt","1/2 teaspoon pepepr","vegetable oil","1 teaspoon cumin seeds","2 tablespoons cumin powder","1 tablespoon chile powder","6 curry leaves","1 tablespoon ginger paste","1/2 cup water","1 tablespoon chile-garlic paste","red food coloring (optional)"],
+    "steps":["Add chicken to a bowl along with the egg, cornstarch, garlic, salt, and pepper. Mix until the chicken is coated and everything is well incorporated. You can choose to mariante the chicken in this micture before frying but it is not required.","Heat some of the vegatable oil in a deep pan or skillet and fry the chicken pieces for about 10 minutes or until fully coocked through.","On another skillet add 2 tablespoons of oil and add the cumin seeds. Cook thill they crackle.","Add the remaining garlic, ginger, cumin, chile powder, pepper, salt, curry leaves, chile-garlic paste, ginger paste, and coloring. Stir well as you add each ingredient. Then add water to thin the paste","Add the fried chick en to the skillet with all the ingredients and cover the chicken completely","All Done! Now you can serve it with some lime if you would like and enjoy!"]
+  },
+
+  {
+    "recipeName":"Strawberry Milk",
+    "link":"strawberrymilk",
```

Comment:
your data looks formatted well, when you move over to the database remember this format 👍